src/tts/magpietts/magpietts.cpp

```
@@ -293,19 +294,25 @@ class MagpieStreamingRuntime::Impl {
     std::unique_ptr<MagpieStreamingWorkspace> workspace;
 };
 
+namespace {
+GgmlLogFilter magpie_ggml_logs;
```

Clang (14.0.6)

[warning] variable 'magpie_ggml_logs' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)